deis-tests-parallel/manifests/deis-tests-parallel-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: deis-tests
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  restartPolicy: Never
+  containers:
+  - name: deis-e2e
+    image: quay.io/deisci/deis-e2e:canary
+    imagePullPolicy: Always
+    command: ["ginkgo"]
+    args: ["-slowSpecThreshold=120.00", "-nodes=30", "-noisyPendings=false", "-progress", "tests/"]
+    env:
+      - name: JUNIT
+        value: "true"
+    volumeMounts:
+    - name: artifact-volume
+      mountPath: /root
+  - name: artifacts
+    image: busybox
+    imagePullPolicy: Always
```

Comment:
Ok, my vote is for pinning to a specific version and specifying `IfNotPresent` for the pull policy. Since `tail` is probably gonna be available on most busybox versions going forward and it's not a big deal to pull such a (relatively) small image, I wouldn't say it's blocking for this PR. Mind creating an issue in this repo to at least investigate making the change though?